README.md

```diff
@@ -26,7 +26,7 @@ The rest of the tools can be installed as follows:
 
 ```console
 gem install mdl
-npm install postcss-cli@7.1.2
+npm install -g postcss-cli@7.1.2
```

Comment:
I'm not sure we want/need to install postcss globally

Reply:
I don't know/especially trust this site but https://flaviocopes.com/npm-packages-local-global/ suggests it's best to keep them locally.

Reply:
Local installation of that package was raising errors during the building with hugo on mac os.
But it could be just my environment badly configured.
Plese ignore the commit if you don't have the package installed globally
( `npm list -g --depth 0` to check)

Reply:
We are installing it locally like in https://github.com/EGI-Foundation/documentation/blob/872de76fa6baa56ca8bdd66622d5d250a55015fb/.travis.yml#L11-L15